docs/manuals/core/config/data-node-config.md

```diff
@@ -136,6 +136,7 @@ described in the previous section [Data node configuration](data-node-config.md)
 
 -   _**exposed_type**_ indicates the data type returned when reading the data node (more examples of reading from CSV data node with different _exposed_type_ is available on [Read / Write a data node](../entities/data-node-mgt.md#csv) documentation):
     -   By default, _exposed_type_ is `"pandas"`, and the data node will read the CSV file as a `pandas.DataFrame`.
+    -   If the _exposed_type_ value provided is `"modin"`, the data node will read the CSV file as a `modin.pandas.DataFrame`.
```

Comment:
Please do not use fixed size font for string. This would be reserved for API elements such as class names.

-> If the \_exposed_type\_ value provided is "modin"

Of course, the same goes for "pandas" or "numpy" around it.